knowledge-base/grid-change-expand-collapse-icons.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Change the expand/collapse icons in a Hierarchical Grid
+description: How to change the expand/collapse icons in a Hierarchical Grid?
+type: how-to
+page_title: Change the expand/collapse icons in a Hierarchical Grid
+slug: grid-kb-change-expand-collapse-icons
+position: 
+tags: telerik,blazor,grid,icon,expand,collapse,change,hierarchical
+ticketid: 1515980
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How to change the expand/collapse icons in a Hierarchical Grid?
+I want to change the built-in plus/minus icons in a Hierarchical Grid. 
+
+## Solution
+You can change the expand/collapse [icons]({%slug general-information/font-icons%}) in the Hierarchical Grid by overriding the built-in plus/minus icons with other icons using custom CSS rules. In addition, you can use the Class parameter of the Grid to add custom CSS Class and modify a specific instance of the Grid, instead of all instances on the page. The code block below demonstrates how to achieve the described approach.
```

Comment:
Consider adding a Notes section at the end to tell people how to get those icons (inspect a rendered icon they want to use and get its content), and to also mention they can use custom icons that are not from the Telerik font (e.g., icons from bootstrap, openiconic and so on) by also specifying the desired font name in addition to the content.

Reply:
Done